collector/physical_disk.go

```diff
@@ -0,0 +1,275 @@
+//go:build windows
+// +build windows
+
+package collector
+
+import (
+	"fmt"
+	"regexp"
+	"strings"
+
+	"github.com/prometheus-community/windows_exporter/log"
+	"github.com/prometheus/client_golang/prometheus"
+	"gopkg.in/alecthomas/kingpin.v2"
+)
+
+func init() {
+	registerCollector("physical_disk", NewPhysicalDiskCollector, "PhysicalDisk")
```

Comment:
The `init()` function will need to be removed, with collector registration details moved to [init.go](https://github.com/prometheus-community/windows_exporter/blob/master/collector/init.go). I'm happy to make this change if you like.

Reply:
Hi Ben, yes please make this change for me if you could.